.github/workflows/build-and-publish-master.yaml

```diff
@@ -10,32 +10,41 @@ jobs:
       - uses: actions/checkout@v2
       - name: Fetch tag history
         run: git fetch --tags
-      - uses: GoogleCloudPlatform/github-actions/setup-gcloud@master
```

Comment:
`GoogleCloudPlatform/github-actions` is deprecated, this moves us to an up-to-date repo.